recipes/supervised/meta.yaml

```diff
@@ -0,0 +1,64 @@
+
+{% set name = "mljar-supervised" %}
+{% set version = "0.10.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mljar-supervised-{{ version }}.tar.gz
+  sha256: 415d32edf48f5cc6a9dfcb1ecd651a6d7df6caa7f8ea5bcdac51fae4231a8f47
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - python ==3.7
```

Comment:
Where is this constraint coming from? The upstream `setup.py` doesn't have a constraint.

Reply:
I removed the `- python ==3.7` line. Can't explain why it was added - I tried many different things to pass checks ...